resources/views/home.blade.php

```diff
@@ -9,7 +9,7 @@
 
                 <div class="panel-body">
                     <ul>
-                        <li>Edit profile</li>
+                        <li><a href='profile'>Edit profile</a></li>
```

Comment:
Do not hard code links.

Use https://laravel.com/docs/5.2/helpers#method-route, e.g. route('member.profile') to get Laravel to generate the correct URI.

If, in future, the route is updated to something else (e.g. /member/profile), as long as the route name is unchanged all links will update automatically.